hpobench/benchmarks/mo/cnn_benchmark.py

```diff
@@ -0,0 +1,520 @@
+    def get_configuration_space(seed: Union[int, None] = None) -> CS.ConfigurationSpace:
+        """Parameter space to be optimized --- contains the hyperparameters
+        """
+        cs = CS.ConfigurationSpace(seed=seed)
+
+        cs.add_hyperparameters([
+            CS.UniformIntegerHyperparameter(
+                'n_conv_layers', default_value=3, lower=1, upper=3, log=False
+            ),
+            CS.UniformIntegerHyperparameter(
+                'conv_layer_0', default_value=128, lower=16, upper=1024, log=True
+            ),
+            CS.UniformIntegerHyperparameter(
+                'conv_layer_1', default_value=128, lower=16, upper=1024, log=True
+            ),
+            CS.UniformIntegerHyperparameter(
+                'conv_layer_2', default_value=128, lower=16, upper=1024, log=True
+            ),
+            CS.UniformIntegerHyperparameter(
+                'n_fc_layers', default_value=3, lower=1, upper=3, log=False
+            ),
+            CS.UniformIntegerHyperparameter(
+                'fc_layer_0', default_value=32, lower=2, upper=512, log=True
+            ),
+            CS.UniformIntegerHyperparameter(
+                'fc_layer_1', default_value=32, lower=2, upper=512, log=True
+            ),
+            CS.UniformIntegerHyperparameter(
+                'fc_layer_2', default_value=32, lower=2, upper=512, log=True
+            ),
+            CS.UniformIntegerHyperparameter(
+                'batch_size', lower=1, upper=512, default_value=128, log=True
+            ),
+            CS.UniformFloatHyperparameter(
+                'learning_rate_init', lower=10 ** -5, upper=1, default_value=10 ** -3, log=True
+            ),
+            CS.CategoricalHyperparameter(
+                'batch_norm', default_value=False, choices=[False, True]
+            ),
+            CS.CategoricalHyperparameter(
+                'global_avg_pooling', default_value=True, choices=[False, True]
+            ),
+            CS.CategoricalHyperparameter(
+                'kernel_size', default_value=5, choices=[7, 5, 3]
+            )
+
+        ])
+        return cs
+
+    @staticmethod
+    def get_objective_names(self) -> List[str]:
+        return ['accuracy', 'model_size']
+
+    @staticmethod
+    def get_fidelity_space(seed: Union[int, None] = None) -> CS.ConfigurationSpace:
+
+        fidelity_space = CS.ConfigurationSpace(seed=seed)
+        fidelity_space.add_hyperparameters([
+            CNNBenchmark._get_fidelity_choices(iter_choice='variable')
+        ])
+        return fidelity_space
+
+    @staticmethod
+    def _get_fidelity_choices(iter_choice: str) -> Tuple[Hyperparameter, Hyperparameter]:
+
+        fidelity1 = dict(
+            fixed=CS.Constant('budget', value=25),
+            variable=CS.UniformIntegerHyperparameter(
+                'budget', lower=1, upper=25, default_value=25, log=False
+            )
+        )
+
+        budget = fidelity1[iter_choice]
+        return budget
+
```

Comment:
We could merge this function with the `get_fidelity_space` function to increase readability.